mscore/musescore.cpp

```diff
@@ -541,8 +541,8 @@ MuseScore::MuseScore()
 #endif
       viewModeCombo->setAccessibleName(tr("View Mode"));
       viewModeCombo->setFixedHeight(preferences.iconHeight + 8);  // hack
-      viewModeCombo->addItem(tr("Page View"));
-      viewModeCombo->addItem(tr("Continuous View"));
+      viewModeCombo->addItem(tr("Page Mode"));
```

Comment:
'View' implies that it doesn't change the score, just how it is presented. This is not the case. But I have to agree, 'Mode' isn't getting this meaning across properly either.